server/models/review-request.js

```diff
@@ -38,6 +38,12 @@ const reviewSchema = new mongoose.Schema({
     type: Date,
     default: Date.now,
   },
+  rating: {
+    type: Number,
+    min: 1,
+    max: 5,
+    default: 5,
+  },
```

Comment:
default rating for both review and user should be 0 or 1 if there are no ratings yet. we can decide if user is able to give 0 rating to the reviewer on the FE when making a review. If not, the min rating that reviewer can set is 1 on the FE.

Reply:
I wasn't sure at the time, and couldn't ask because of the timezone differences. I will update the code accordingly.